examples/python/Pipfile

```diff
@@ -12,6 +12,7 @@ verify_ssl = true
 
 [packages]
 looker-sdk = "*"
+#looker-sdk = {version="0.1.3b18", index="testpypi"}
```

Comment:
been meaning to leave this in for a while, makes testing the "canary" version on testpypi easier